src/kOS.Safe/Encapsulation/SuffixedList.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
```

Comment:
The reason i went on this tear was because I found yet another place were we were returning a KSP type that was useless to the user but because it was wrapped up in the ListValue you couldnt see it.

If i could have said "ISuffixed and primitives and strings" i likely would have. Dispensing with the other two list types.